core-schema/src/main/resources/sql-schema/V6__Mod_kitpvp.sql

```diff
@@ -1,20 +1,30 @@
 
 CREATE TABLE kitpvp_statistics (
   user_id INT PRIMARY KEY,
-  kills INT NOT NULL,
-  deaths INT NOT NULL,
-  assists INT NOT NULL,
-  current_killstreak INT NOT NULL,
-  highest_killstreak INT NOT NULL,
-  experience INT NOT NULL,
-  bounty INT NOT NULL,
-
-  FOREIGN KEY (user_id) REFERENCES user_ids (id) ON DELETE CASCADE
+  kills INT NOT NULL DEFAULT 0,
+  deaths INT NOT NULL DEFAULT 0,
+  assists INT NOT NULL DEFAULT 0,
+  current_killstreak INT NOT NULL DEFAULT 0,
+  highest_killstreak INT NOT NULL DEFAULT 0,
+  experience INT NOT NULL DEFAULT 0,
+  bounty INT NOT NULL DEFAULT 0,
+
+  FOREIGN KEY (user_id) REFERENCES user_ids (id) ON DELETE CASCADE,
+  INDEX kills_index (kills),
+  INDEX highest_killstreak_index (highest_killstreak),
+  CONSTRAINT nonnegative_kills CHECK (kills >= 0),
```

Comment:
Is this necessary?

Reply:
You mean the CHECK for non-negative kills? I don't see any reason *not* to include it. It improves data integrity somewhat.